src/main/java/org/kiwiproject/dropwizard/util/health/MongoHealthCheck.java

```diff
@@ -28,28 +28,29 @@ public MongoHealthCheck(DB db) {
     @SuppressWarnings({"deprecation", "ConstantConditions"})
     @Override
     protected Result check() throws Exception {
-        try {
-            var host = db.getMongo().getAddress().toString();
-            var dbName = db.getName();
+        var host = db.getMongo().getAddress().toString();
+        var dbName = db.getName();
 
-            var result = db.getStats();
+        try {
+            var statsCommandResult = db.getStats();
```

Comment:
Suggest maybe just `stats`?

Reply:
But we call `.ok()` on it which is a method about the result of the command.  This is another thing missing in `MongoClient`

Reply:
OK then how about just `statsResult`? Shorter but still gets the point across